scripts/legacy_expression_tool.md

```diff
@@ -0,0 +1,171 @@
+# Legacy Expression tool
+
+With release 3.2.0 of IoT Agent Node lib, the legacy expressions language support 
+has been removed. This folder contains the scripts to ease to migrate the legacy expressions. 
+
+A manual migration is required, because the expressions need to be converted to JEXL expressions manually.
+
+This python script will help you to migrate the legacy expressions to JEXL expressions. It will search for the legacy 
+expressions in the database and
+replace them with the new expressions.
+
+This script will search for the legacy expressions in the database and replace them with the new expressions. It can be also 
+be executed in dry-run mode, so it will not modify the database, but it will generate a series os files that can be used to 
+ease and verify data migration.
+
+## Output files
+
+The script generates 4 different files each time it is executed: 
+
+### `legacy_expression_ocurrences.json`
+
+This file contains information about the objects where the legacy expressions are found. It contains the following information 
+for each legacy expression found:
+
+- _id: The id of the object where the legacy expression is found
+- expression: The legacy expression found
+- type: The type of the object where the legacy expression is found (I.E: active.expression)
+- fiware_service: The fiware service of the object where the legacy expression is found
+- fiware_servicepath: The fiware service path of the object where the legacy expression is found
+
+### `legacy_expression_list.json`
+
+This file contains just a list of all the legacy expressions found in the database. It can be used to build the translation 
+file. The file looks like this:
+
+```json
+[
+    "${legacy-expression-1}",
+    "${legacy-expression-2}",
+    ...
+    "${legacy-expression-n}"
+]
+```
+
+### `objects_replaced.json`
+
+This file contains a list of objects where the legacy expressions have been replaced with all the information associated. This 
```

Comment:
```suggestion
This file contains a list of documents where the legacy expressions are going to be replaced with all the information associated. This 
```

I think is more precise this way, as in the default mode (dry run) no document is replaced.

In addition, the term "document" is more precise when we refer to documents stored in MongoDB. Of course, they are also JSON objects, but JSON object could refer to other things.